src/model/tflite/generated/mod.rs

```diff
@@ -0,0 +1,19 @@
+// NOTE: the files `image_segmenter_metadata_schema_generated.rs`, `metadata_schema_generated.rs`,
```

Comment:
I would like to know the intention of adding generated files. These are generated by `scripts/flatbuffers-init.sh`. Do we want users to generate them? Or would we ask users to use our generated version?

Reply:
> I would like to know the intention of adding generated files. These are generated by `scripts/flatbuffers-init.sh`.

The generated files are used to parse the TfLite model file and get metadata from it, such as input image shape, classification labels, vocabs.txt, etc.   

> Do we want users to generate them? Or would we ask users to use our generated version?

No. The library developers generate them once and ask users to use the generated version.

The `scripts/flatbuffers-init.sh` is used to follow the update upstream.

Reply:
Got it.